tests/test_import_offline_snapshot.rs

```diff
@@ -60,3 +76,39 @@ async fn test_import_offline_snapshot() {
         }
     }
 }
+
+async fn populate_postgres(docker: &clients::Cli, state: InMemoryPermanentStorageState) -> (Container<'_, PostgresImage>, Postgres) {
+    // init docker container (this should be extract to a reusable module if going to be used in other tests)
+    let pg_image = RunnableImage::from(PostgresImage::default().with_user("postgres").with_password("123").with_db_name("stratus"))
+        .with_mapped_port((5432, 5432))
+        .with_volume(("./static/schema/001-init.sql", "/docker-entrypoint-initdb.d/001-schema.sql"))
+        .with_volume(("./static/schema/002-schema-external-rpc.sql", "/docker-entrypoint-initdb.d/002-schema.sql"))
+        .with_tag("16.1");
+    let pg_container = docker.run(pg_image);
+
+    // init postgres client
+    let pg = Postgres::new("postgres://postgres:123@localhost:5432/stratus").await.unwrap();
```

Comment:
use clap?

Reply:
No need for clap here, unless we want to connect to some external database.

The connection URL is hardcoded here because it is using the same parameters defined in the test-container, so assuming this test only wants to use the test container, the URL wil never be different.